modules/migration-network-traffic-redirection-strategies.adoc

```diff
@@ -0,0 +1,49 @@
+// Module included in the following assemblies:
+//
+// * migrating_from_ocp_3_to_4/planning-considerations-3-4.adoc
+
+[id="migration-network-traffic-redirection-strategies_{context}"]
+= Network traffic redirection strategies
+
+After a successful migration, you have to redirect network traffic of your stateless applications from the source cluster to the target {product-title} {product-version} cluster.
+
+All strategies for redirecting network traffic described below are based on these assumptions:
+
+* Applications are deployed on the {product-title} {product-version} cluster. The original application pods are still running on the source cluster.
+* Each relevant application has a route that contains the source cluster hostname.
```

Comment:
If "relevant" is important, please describe to our customer how to distinguish between relevant applications and applications that are not relevant.  If "relevant" isn't important, please remove the word.

Reply:
"Relevant" can be removed. Each application route contains the source cluster hostname.